.vscode/settings.json

```diff
@@ -1,6 +0,0 @@
-{
-  "files.associations": {
-    "*.json": "jsonc"
-  },
-  "graphql-config.load.rootDir": "packages/apps/graph"
-}
```

Comment:
I'm not entirely sure this file should be deleted. @alber70g , I was under the impression it was needed?